client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -163,19 +164,27 @@ export class List extends React.Component {
 			);
 		}
 
-		const headerText = this.props.translate( 'Domains', { context: 'A navigation label.' } );
-
 		/* eslint-disable wpcalypso/jsx-classname-namespace */
 		return (
 			<Main wideLayout>
-				<DocumentHead title={ headerText } />
+				<DocumentHead
+					title={ this.props.translate( 'Domains', { context: 'A navigation label.' } ) }
+				/>
 				<SidebarNavigation />
 				<PlansNavigation cart={ this.props.cart } path={ this.props.context.path } />
 				{ this.domainWarnings() }
 
 				{ this.domainCreditsInfoNotice() }
 
-				<SectionHeader label={ headerText }>{ this.headerButtons() }</SectionHeader>
+				<SectionHeader
+					label={ this.props.translate( 'Domains associated with %(siteTitle)s', {
```

Comment:
Is it possible for site title to be blank here? If so, should we handle that with a generic phrase like 'Domains associated with your site'?

Reply:
It seems not, and that in such cases, the fallback (both in the sidebar and presumably elsewhere in Calypso) is the site title. Either way, I think that's more beneficial in solving the issue of clarity rather than stating "your site". :)

<img width="1466" alt="Screenshot 2019-11-04 at 22 44 59" src="https://user-images.githubusercontent.com/43215253/68164529-e670d200-ff54-11e9-93cf-06d46abe1ef1.png">

Reply:
Indeed :) Thanks for following up!